blueprints/engine-blueprint/index.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+const path = require('path');
+const MainBlueprint = require('../../../index');
+
+/*
+  Create an "addon blueprint" that simply defers to our
+  top level entry point as the blueprint.
+  This is basically just a work around for
+  https://github.com/ember-cli/ember-cli/issues/6952.
+  Once that issue is fixed and released we can remove:
+    * ember-addon keyword in package.json
+    * ember-addon key in package.json
+    * ember-addon-main.js file
+    * blueprints/ folder
+ */
+module.exports = Object.assign({}, MainBlueprint, {
+  init() {
+    this._super.init.apply(this, arguments);
+
+    this.path = path.join(__dirname, '..', '..', '..');
+    this.name = 'ember-engines/engine-blueprint';
```

Comment:
I think this is all just work-around code that will be removed when the mentioned issue is resolved and this file is removed.

Though, I just copied this from the glimmer blueprint so I could be wrong. I guess @rwjblue will be able to answer if this should be moved or not.